### ISSUE TITLE
[Effect Suggestion] Dude Where's My Car

### ISSUE THREAD

**Issue by htorw** [gta-chaos-mod/ChaosModV#796]
**Name of effect**
Dude Where's My Car

**Duration if timed effect (long or short timed?)**
Short

**Effect Description**
Every time player tries to enter a car it gets teleported away

**Comment by anto4440** [comment]
So forcefield but only for cars? or 'lock all vehicles'? Or #794 combined with 'remove current vehicle?

**Comment by htorw** [comment]
Yes every time the player tries to enter a car it gets despawned or teleported away
